js/helpers.js

```diff
@@ -222,7 +222,7 @@ function restoreStatus() {
         requirementDropdown.value = elective.date;
 
         requirementEntry.appendChild(requirementLabel);
-        requirementEntry.appendChild(document.createElement('br'));
+       // requirementEntry.appendChild(document.createElement('br'));
```

Comment:
If we don't need this, just go ahead and delete it -- no need to comment code unless it's going to be used later.